x-pack/legacy/plugins/siem/public/components/ml_popover/ml_popover.tsx

```diff
@@ -92,26 +81,40 @@ const initialState: State = {
   refreshToggle: true,
 };
 
+const defaultFilterProps: JobsFilters = {
+  filterQuery: '',
+  showCustomJobs: false,
+  showElasticJobs: false,
+  selectedGroups: [],
+};
+
 export const MlPopover = React.memo(() => {
   const [{ refreshToggle }, dispatch] = useReducer(mlPopoverReducer, initialState);
 
   const [isPopoverOpen, setIsPopoverOpen] = useState(false);
-  const [showCustomJobs, setShowCustomJobs] = useState(false);
-  const [showElasticJobs, setShowElasticJobs] = useState(false);
-  const [isLoadingJobSummaryData, jobSummaryData] = useJobSummaryData([], refreshToggle);
-  const [isCreatingJobs, setIsCreatingJobs] = useState(false);
-  const [filterQuery, setFilterQuery] = useState('');
+  const [filterProperties, setFilterProperties] = useState<JobsFilters>(defaultFilterProps);
```

Comment:
You can use just this form:

```suggestion
const [filterProperties, setFilterProperties] = useState(defaultFilterProps);
```

Since defaultFilterProps is a JobsFilters already and your types will still be fine.